core/platform/bulk_email/mailchimp_bulk_email_services.py

```diff
@@ -252,18 +252,20 @@ def add_or_update_user_status(
             'email_address': user_email,
             'status': 'subscribed',
             'tags': [tag],
-            'merge_fields': {
-                'NAME': merge_fields['NAME']
-            }
+            'merge_fields': {}
         }
         subscribed_mailchimp_data = {
             'email_address': user_email,
             'status': 'subscribed',
-            'merge_fields': {
-                'NAME': merge_fields['NAME']
-            }
+            'merge_fields': {}
         }
 
+        if merge_fields['NAME'] is not None:
```

Comment:
Wouldn't this result in a KeyError if there is no NAME key in merge_fields?

Reply:
Absolutely, Thank goodness you caught that! It was a rather silly mistake on my part. Thank you very much.
I have made the necessary change, PTAL